components/script/dom/document.rs

```diff
@@ -2355,6 +2355,13 @@ impl Document {
             entry.hint |= RESTYLE_STYLE_ATTRIBUTE;
         }
 
+        // FIXME(emilio): This should become something like
+        // element.is_attribute_mapped(attr.local_name()).
+        if attr.local_name() == &local_name!("width") ||
+           attr.local_name() == &local_name!("height") {
+            entry.hint |= RESTYLE_SELF;
+       }
```

Comment:
Good catch, want to write a PR to fix it? Or should I go ahead and do it? I filed https://github.com/servo/servo/issues/16486 for a lint that would've catched this.